### PR TITLE
Autopep8 taskserver.py

### PR DESCRIPTION
**Pull request by Krigpl**
Using `yapf`

### CODE REVIEW

File: golem/task/taskserver.py [hunk 1]
```diff
@@ -942,9 +983,13 @@ def _find_sessions(self, subtask):
...
 
-    def _send_waiting(self, elems_set, subtask_id_getter, req_type, session_cbk, p2p_node_getter):
+    def _send_waiting(self, elems_set, subtask_id_getter, req_type,
+                      session_cbk, p2p_node_getter):
         for elem in elems_set.copy():
-            if hasattr(elem, '_last_try') and (datetime.datetime.now() - elem._last_try) < datetime.timedelta(seconds=30):
+            if hasattr(
+                    elem,
+                    '_last_try') and (datetime.datetime.now() - elem._last_try
+                                      ) < datetime.timedelta(seconds=30):
```
Comment (jiivan) [inline golem/task/taskserver.py]: :eye: It's a matter of taste but it doesn't look readable to me.

**Review by jiivan — APPROVED** [review]
LGTM, only this `and` formatting distastes me :(

File: golem/task/taskserver.py (hunk 1 above)
Comment (Krigpl) [reply golem/task/taskserver.py]: Oh, yeah, definitely. I'll `# noqa` it :P

**Comment by jiivan** [comment]
Shippable failed. Is this caused by ignoring noqa comments or something else?

**Comment by Krigpl** [comment]
> Shippable failed. Is this caused by ignoring noqa comments or something else?

That was because of the long `import` statements, hopefully this is fixed now.

**Pull request merged**